backend/controller/controller.go

```diff
@@ -1453,13 +1453,14 @@ func (s *Service) terminateRandomRunner(ctx context.Context, key model.Deploymen
 func (s *Service) deploy(ctx context.Context, reconcile model.Deployment) error {
 	client, err := s.reserveRunner(ctx, reconcile)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to reserve runner for %s %s: %w", reconcile.Key, reconcile.Module, err)
 	}
 
 	_, err = client.runner.Deploy(ctx, connect.NewRequest(&ftlv1.DeployRequest{DeploymentKey: reconcile.Key.String()}))
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to request deploy %s %s: %w", reconcile.Key, reconcile.Module, err)
```

Comment:
I think module is redundant here because each deployment key is `dpl-modulename-randombytes`